Juriba.Platform/Juriba.Platform.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule        = 'Juriba.Platform.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.0.38.0'
+    ModuleVersion     = '0.0.38.1'
```

Comment:
Changes to scripts in the examples folder do not need the PowerShell module to be republished. It's not necessary to increment the module version.

Reply:
How do I roll it back? It's my first time so I'm not sure what to do